src/lib/driver/message_handler.ts

```diff
@@ -83,7 +82,8 @@ export class DriverMessageHandler {
         return {};
       }
       case DriverCommand.enableErrorReporting: {
-        driver.enableErrorReporting();
+        // This capability no longer exists but we keep the command here for backwards
```

Comment:
should we throw an error here to match what we are doing with firmware updates? I think it's probably better to so that we are clear to the client what's happening